[DOCS] Dashboard API breaking changes from #41125

rudolf opened the pull request:
## Summary

Adds the breaking changes from #41125 to the 7.3 migration doc.

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)

### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)
- [ ] This includes a feature addition or change that requires a release note and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)

elasticmachine commented:
Pinging @elastic/kibana-docs

KOTungseth commented on docs/migration/migrate_7_3.asciidoc:
```diff
@@ -0,0 +1,32 @@
+
+See also <<release-highlights>> and <<release-notes>>.
+
+* <<breaking_73_dashboard_import_export>>
```
Because there is only one list item, and users don't need to scroll to view the text, we probably don't need this bullet.

KOTungseth commented on docs/migration/migrate_7_3.asciidoc:
```diff
@@ -0,0 +1,32 @@
+
+[float]
+[[breaking_73_dashboard_import_export]]
+=== Dashboards Import/Export API
```
To match the [other documentation](http://localhost:8000/guide/dashboard-import-api.html), I would change this to `Dashboard import and export APIs`.

KOTungseth commented on docs/migration/migrate_7_3.asciidoc:
```diff
@@ -0,0 +1,32 @@
+=== Dashboards Import/Export API
+
+[float]
+==== Dashboards Import/Export API error status codes
```
I would change this to `Dashboard import and export APIs error status codes`.

KOTungseth commented on docs/migration/migrate_7_3.asciidoc:
```diff
@@ -0,0 +1,32 @@
+
+[float]
+==== Dashboards Import/Export API error status codes
+*Details:* The Dashboards Import/Export API will return more accurate HTTP
```
In this instance, it should be `dashboard import and export APIs`.

KOTungseth commented on docs/migration/migrate_7_3.asciidoc:
```diff
@@ -0,0 +1,32 @@
+HTTP 403 when the authenticated user lacks sufficient privileges.
+
+*Impact:* Review the error handling logic of any integrations with the
+Dashboards Import/Export API.
```
Change this to `dashboard import and export APIs`.

rudolf commented:
@gchaps I wasn't thinking, I've fixed the base branch to point to 7.x and will backport this to 7.4 and 7.3

@KOTungseth Thanks, I've updated the copy.
